### DISCUSSION TITLE
Questions about the libRomanBeta

### DISCUSSION THREAD

**Discussion by PlantNightShade** [thatskymod/Sky-CotL-Scripts#36] [General]
I wanted to know for the lib (libRomanBeta) does it give you the beta version game or only for the use of the beta game

**Comment by thatskymod** [comment]
LibRomanBeta was specifically designed to use with the Beta version of the game. It **does not** grant you access to the Early Access /Testing program of Sky.

**Comment by Stardustcalisto** [comment]
I don't know how to activate the beta secret command
